lib/Jsrt/Jsrt.cpp

```diff
@@ -1439,7 +1399,7 @@ CHAKRA_API JsDefineProperty(_In_ JsValueRef object, _In_ JsPropertyIdRef propert
 
 CHAKRA_API JsCreateArray(_In_ unsigned int length, _Out_ JsValueRef *result)
 {
-    return ContextAPIWrapper<true>([&] (Js::ScriptContext *scriptContext) -> JsErrorCode {
+    return ContextAPINoScriptWrapper([&](Js::ScriptContext *scriptContext) -> JsErrorCode {
```

Comment:
No, it is not unrelated. JsScriptHost onload event may have called when the script execution is active. Updated these 2 similar to String interface.